.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
```

Comment:
Make it a bit more specific:
```suggestion
    name: QA / ${{ matrix.toxenv }}
```

Reply:
<img width="410" alt="image" src="https://user-images.githubusercontent.com/7377671/80892462-846f0a80-8cf4-11ea-9cc8-9bae43e43c60.png">

Isn't it specific?

Reply:
I've put just toxenv to the name in 988dfbc. Looks good IMO.